src/main/java/net/objecthunter/exp4j/function/Functions.java

```diff
@@ -41,11 +41,9 @@
     private static final int INDEX_LOG10 = 19;
     private static final int INDEX_LOG2 = 20;
     private static final int INDEX_SGN = 21;
-    private static final int INDEX_SIN_DEGREES = 22;
-    private static final int INDEX_COS_DEGREES = 23;
-    private static final int INDEX_TAN_DEGREES = 24;
+    private static final int INDEX_deg2rad = 22;
```

Comment:
First let me mention that as a third party that I can't neither accept nor reject a pull request, but I'll give you some _pointers_: 
- (mandatory) Whenever sending a patch (to anyone) you need to respect the coding style
- (partly optional) In a test-based library like `exp4j` one expects some tests for the code (at least some sanity check)
- (optional) since there's no way of reverting a function like `deg2rad` you might want to add something like `rad2deg` otherwise you wouldn't be able to use inverse trigonometric functions (as discussed in #72)